frontend/plan/components/modals/CalendarModal.tsx

```diff
@@ -1,137 +1,156 @@
 import React, { useEffect, useState } from "react";
-import { toast } from "react-toastify";
 import styled from "styled-components";
+import { connect } from "react-redux";
 
-interface RowProps {
-    align?: "flex-start" | "flex-end" | "center" | "baseline" | "stretch";
-}
+const Outer = styled.div`
+    overflow-x: hidden;
+`;
 
-const Row = styled.div<RowProps>`
+const Row = styled.div`
     display: flex;
-    flex-wrap: wrap;
-    align-items: ${(props) => props.align || "stretch"};
+    align-items: center;
 `;
 
-interface ColProps {
-    span?: number;
-}
-
-const Outer = styled.div`
-    overflow-x: hidden;
+const Col = styled.div`
+    flex: 1;
+    padding: 0;
 `;
 
-const Col = styled.div<ColProps>`
-    flex: ${(props) => props.span || 1};
-    padding: 0;
-    box-sizing: border-box;
+const BigText = styled.p`
+    font-size: 14px;
+    line-height: 1.5;
 `;
 
-const LoveFromLabs = styled.div`
+const Button = styled.a`
+    height: 30px;
     display: flex;
     align-items: center;
     justify-content: center;
+    color: white;
+    margin: 0 5px;
+    border-radius: 5px;
 `;
 
-const BigText = styled.p`
-    font-size: 14px;
-    line-height: 1.5;
+const ICSButton = styled(Button)`
+    background-color: #07aced;
+`;

```

Comment:
Is there a reason why the ICS URL label is a button? We should change this to a text